cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3001"
```

Comment:
I think this should be the heroku link, or the local host link for the api, localhost:3000

Reply:
After talking to team, realised I misunderstood this so resolving this comment